universal-application-tool-0.0.1/test/support/TestQuestionBank.java

```diff
@@ -107,6 +106,21 @@ private static Question applicantFavoriteColor(QuestionEnum ignore) {
     return maybeSave(definition);
   }
 
+  private static Question applicantHouseholdMembers(QuestionEnum ignore) {
+    QuestionDefinition definition =
+        new RepeaterQuestionDefinition(
+            VERSION,
+            "applicant household members",
+            Path.create("applicant.household_members"),
```

Comment:
(blocking, maybe?) I _think_ the repeater question path will need to include square brackets (`applicant.household_members[]`). This is yet another reason why `Path` shouldn't be a constructor argument. 

We may end up not needing the square brackets but they're being added to repeater questions, so this question should probably have it to match the real repeater questions.

Reply:
Done.